src/js/jsx/sections/export/LayerExports.jsx

```diff
@@ -45,48 +46,30 @@ define(function (require, exports, module) {
      * @private
      * @type {Immutable.OrderedMap.<string, {id: string, title: string}>}
      */
-    var _scaleOptions = Immutable.OrderedMap({
-        "0.5": {
-            id: "0.5",
-            title: "0.5x"
-        },
-        "1": {
-            id: "1",
-            title: "1x"
-        },
-        "1.5": {
-            id: "1.5",
-            title: "1.5x"
-        },
-        "2": {
-            id: "2",
-            title: "2x"
-        }
-    });
+    var _scaleOptions = ExportAsset.SCALES
+        .toSeq()
+        .reduce(function (map, scale) {
+            var obj = {
+                id: scale.toString(),
+                title: scale.toString()
+            };
+            return map.set(scale.toString(), obj);
+        }, Immutable.OrderedMap());
 
     /**
      * The options for the format datalist
      * @private
      * @type {Immutable.OrderedMap.<string, {id: string, title: string}>}
      */
-    var _formatOptions = Immutable.OrderedMap({
-        "png": {
-            id: "png",
-            title: "PNG"
-        },
-        "jpg": {
-            id: "jpg",
-            title: "JPG"
-        },
-        "svg": {
-            id: "svg",
-            title: "SVG"
-        },
-        "pdf": {
-            id: "pdf",
-            title: "PDF"
-        }
-    });
+    var _formatOptions = ExportAsset.FORMATS
```

Comment:
More Seq questions here. Keep in mind that if you use this data structure as a lookup table then every time you perform a lookup you have to re-do the `reduce` part!